teams/delius-iaps/components/iaps_server/delius_iaps_install_ndelius_interface.yml

```diff
@@ -0,0 +1,474 @@
+---
+name: delius_iaps_install_ndelius_interface
+description: Installs delius_iaps_install_ndelius_interface
+schemaVersion: 1.0
+parameters:
+  - Version:
+      type: string
+      default: 0.1.0
+      description: Component version
+  - Platform:
+      type: string
+      default: "Windows"
+      description: Platform name
+  - S3ArtefactBucket:
+      type: string
+      description: Name of s3 bucket holding software artefacts
+phases:
+  - name: build
+    steps:
+      - name: DownloadArtefacts
+        action: ExecutePowerShell
+        inputs:
+          commands:
+            - |
+              $ErrorActionPreference = "Stop"
+              $VerbosePreference = "Continue"
+
+              Read-S3Object -BucketName {{ S3ArtefactBucket }} -KeyPrefix "delius-iaps/apps/ndelius-interface/" -Folder " ${env:TEMP}\ndelius-interface"
+      - name: InstallNDeliusInterface
+        action: ExecutePowerShell
+        inputs:
+          commands:
+            - |
+              $ErrorActionPreference = "Stop"
+              $VerbosePreference = "Continue"
+
+              Start-Process -Wait -FilePath "${env:TEMP}\ndelius-interface\setup.exe" -ArgumentList "/quiet /qn" -Verb RunAs
+              New-Item -ItemType Junction -Path "C:\Program Files\I2N\IapsNDeliusInterface" -Target "C:\Program Files (x86)\I2N\IapsNDeliusInterface"
+      - name: RemoveInvalidShortcut
+        action: ExecutePowerShell
+        inputs:
+          commands:
+            - |
+              Write-Host('Remove invalid shortcut')
+              Remove-Item -Path "C:\Users\Public\Desktop\Migration Utility (Iaps-NDelius IF).lnk"
+
+              Write-Host('Grant local Users group full access recursively to I2N program dir')
+              $i2npath = "C:\Program Files (x86)\I2N"
+              $acl = (Get-Item $i2npath).GetAccessControl('Access')
+              $rule = New-Object System.Security.AccessControl.FileSystemAccessRule("Users", "FullControl", "ContainerInherit,ObjectInherit", "None", "Allow")
+              $acl.SetAccessRule($rule)
+              Set-Acl $i2npath $acl
+
+              Write-Host('Remove invalid shortcut')
+              if ( Test-Path "C:\Users\Public\Desktop\Migration Utility (Iaps-NDelius IF).lnk") { Remove-Item -Path "C:\Users\Public\Desktop\Migration Utility (Iaps-NDelius IF).lnk" }
```

Comment:
This line and line 45 seem to be doing the same action. one of them should be removed

Reply:
redundant line removed